cfgov/paying_for_college/tests/test_models.py

```diff
@@ -81,16 +83,16 @@ def test_loan_quiz_template(self):
             self.loan_quiz_page.get_template(HttpRequest()),
             'paying-for-college/choose-a-student-loan.html')
 
-    def test_debt_page_template(self):
-        self.assertEqual(
-            self.debt_page.get_template(HttpRequest()),
-            'paying-for-college/repaying-student-debt.html')
-
     def test_college_costs_template(self):
         self.assertEqual(
             self.college_costs_page.get_template(HttpRequest()),
             'paying-for-college/college-costs.html')
 
+    def test_debt_page_template(self):
```

Comment:
Snce the templates are now all hardcoded since there are different page types, are these tests testing anything significant beyond whether Wagtail's `get_template` method works right?

Reply:
Yes, the template tests are not very valuable at this point.

Reply:
You could move at least some of these (except for StudentLoanQuizPage, which has the block ID logic) from `get_template()` methods to a `template` property on the model:

```py
class CollegeCostsPage(PayingForCollegePage):
    template = 'paying-for-college/college-costs.html'
```

This takes care of the coverage. 😀

Reply:
My tunnel vision is showing. Thank you!